test/metabase/models/interface_test.clj

```diff
@@ -130,3 +133,11 @@
                (migrate {:pie.show_legend          legend
                          :pie.show_legend_perecent percent
                          :pie.show_data_labels     labels})))))))
+
+(deftest encrypted-data-with-no-secret-test
+  (is (= {:a 1}
+         (mi/encrypted-json-out "{\"a\": 1}")))
```

Comment:
I can't tell what the behaviour is you're trying to test is in both these tests.

Without `testing` descriptions, we have to do mind-reading when reading tests to figure out what exactly is being covered, or extra work to guess it from the code.

Reply:
Oh, right, I went through too many ideas with this change and this test remained in the worst condition. Adding description now...

Reply:
Looks good now, thanks